api-tests/specs/practitioner-service/linkperson.js

```diff
@@ -0,0 +1,83 @@
+const requests = require('../../data/practitioner-service/linkperson')
+const conf = require('../../config/conf')
+const practitionerEndpointUri = ':8081/practitioner'
+let linkpersonEndpointUri = ':8081/practitioner/{personId}/link'
+
+
+describe('As a user with Update Person permission I want to link a Person to an existing User So that they have the Persons roles when they login to the system', function () {
+
+    it.only('Given a Person exists who is not linked to a User AND a User exists that is not linked to a Person When I submit an API request to link a Person who is NOT linked to a User to a User who is not linked to a Person in this trial Then the Person and User are linked and I receive a success acknowledgement ', async () => {
```

Comment:
Please delete the .only tag from the mocha scenario, we use it only for debugging purpose. It applies to all the scenarios.